src/main/resources/resources/logisim/strings/gui/gui.properties

```diff
@@ -660,7 +660,7 @@ argSubOption = Substitutes library lib1 with lib2. Arguments: lib1 lib2.
 argSubOptionArgName = args
 argTemplateOption = Select template. Argument: "empty" for an empty template, "plain" for the default template, or the filename of the template to load.
 argTemplateOptionArgName = arg
-argTestCircGenOption = Open up the circ file and write a new one to later check the XML content. Arguments: <circ_input> <circ_output>
+argTestCircGenOption = Open a .circ file of any version of logisim and write a new one in the current format used in logisim-evolution. Arguments: <circ_input> <circ_output>
```

Comment:
Shouldn't we spell `Logisim` with capital `L`? Also you used both `Logisim` and `Logisim-evolution`. Not sure if that can be confusing.

Reply:
updated the l -> L. The confusion is minimal I think, so will not change that